pkgs/development/python-modules/databricks-connect/default.nix

```diff
@@ -0,0 +1,31 @@
+{ lib, isPy37, pkgs, buildPythonPackage, fetchPypi, six, py4j }:
+
+buildPythonPackage rec {
+  pname = "databricks-connect";
+  version = "7.1.0";
+  sha256 = "996a9d0f271f6c7edbd2d85b2efb6ff4e58d15222e80f87ca17fdbf224e17056";
+
+  disabled = !isPy37;
```

Comment:
From looking on pypi, it looks like this package supports a pretty wide range of Python versions - do you even need this disabled flag? It actually seems to support every version in Nixpkgs - though if you want to limit it to isPy3, that would be reasonable too.

Reply:
Have now removed py37 constraint.